### DISCUSSION TITLE
Applying for Maintainer

### DISCUSSION THREAD

**Discussion by marcusleonas** [devicons/devicon#1914] [General]
## ABOUT ME
Hiya, I'm Marcus! I'm a frontend web developer and theatre technician from the UK!

## ME AND DEVICONS
I have been using Adobe Illustrator and Photoshop for over 3 years and JS over 4 years, and I'm currently learning TypeScript. I have been wanting to add a few icons to devicons for some time now.

## WHY I WANT TO JOIN
I need to learn git, and i've been wanting to add a few icons for a while now.

## MY SOCIALS
https://mas.to/@marcusleonas
https://instagram.com/marcusleonas_public
https://www.linkedin.com/in/marcusleonas

**Comment by Snailedlt** [comment]
Hi there Marcus! 👋
We would love to have someone who knows and works with design tools on our team, but a general rule of thumb we don't accept maintainers unless they've already contributed to devicons.

Good news though! You don't need to be a maintainer to contribute! 🙌 
Feel free to contribute with some icons by following the instructions in our [wiki](https://github.com/devicons/devicon/wiki). If your contributions are good we'll reconsider your application. Thanks in advance and dolet us know if you need any help! If your !

**Reply by Snailedlt** [reply]
Ohh, and please comment on our existing discussion when applying in the future :)
#1436
